docs/epiviz/file_locations.rst

```diff
@@ -0,0 +1,27 @@
+.. _file-locations:
+
+File Locations on the Cluster
+=============================
+
+ *  ``/ihme/epi/dismod_at``
+
+    *  ``share/local_odbc.ini`` - This is an ODBC file to use when running on
+       the Fair cluster. This is used in ``dismodel_main.py`` while
+       their password technique involves using the unavailable J drive.
+
+    *  ``bin/`` - Contains tools for modelers to manipulate bundles on
```

Comment:
If we're formalizing this :hankey: we should move the elmo environment out of my personal directory.

Reply:
That might be work. The hacked Elmo is what we have now.

Reply:
Oh, I'm not suggesting we move away from the hacked elmo just that it shouldn't live in a place with my username on it.